tools/pd-ut/ut.go

```diff
@@ -746,13 +746,13 @@ func buildTestBinaryMulti(pkgs []string) ([]byte, error) {
 
 	// go test --exec=xprog --tags=tso_function_test,deadlock -vet=off --count=0 $(pkgs)
 	// workPath just like `/pd/tests/integrations`
-	xprogPath := path.Join(workDir, "bin/xprog")
+	xprogPath := filepath.Join(workDir, "bin/xprog")
```

Comment:
If we consider different operating systems, maybe`bin/xprog` also need to be changed?